browser-test/src/admin_predicates.test.ts

```diff
@@ -371,13 +371,15 @@ describe('create and edit predicates', () => {
     await applicantQuestions.clickNext()
 
     // Earlier than 2021-01-01 is allowed
-    // TODO(#3859): 2021-01-01 evaluates as earlier, but it shouldn't.
     await applicantQuestions.answerDateQuestion('2021-01-02')
     await applicantQuestions.clickNext()
     await applicantQuestions.expectReviewPage()
     await page.goBack()
     await applicantQuestions.answerDateQuestion('2020-12-31')
     await applicantQuestions.clickNext()
+    await page.goBack()
+    await applicantQuestions.answerDateQuestion('2020-01-01')
```

Comment:
This is in effect double testing the condition which we aren't making a point of doing in this test.  I tink you want to remove this and change the 2021-01-02 above (the false evaluation) to 2021-01-01